aurora/src/components/contact/ContactIcon.tsx

```diff
@@ -1,5 +1,27 @@
+import Image from 'next/image';
+import styled from 'styled-components';
 import { ContactIconInterface } from './Contact';
 
-export default function ({ icon }: { icon: ContactIconInterface }) {
-    return <div>{icon.name}</div>;
+export default function ContactIcon ({ icon }: { icon: ContactIconInterface }) {
+    return (
+        <ImageStyle href={icon.href} target="_blank">
+            <Image
```

Comment:
As an `Image`, this is fine, yea, NextJS has their own super-optimized tag for that